rtgui/filecatalog.cc

```diff
@@ -1808,6 +1808,16 @@ void FileCatalog::reparseDirectory ()
         _refreshProgressBar();
     }
 
+    // when handling FILE_MONITOR_EVENT_DELETED we can stop here, otherwise there can be significant performance penalty
+    // with network shares
+    if (onlyDelete && fileNamesToRemove.size() == 1) {
+        auto pos = std::find(fileNameList.cbegin(), fileNameList.cend(), *fileNamesToRemove.cbegin());
```

Comment:
Can `*fileNamesToRemove.cbegin()` be simpilified?

```suggestion
        auto pos = std::find(fileNameList.cbegin(), fileNameList.cend(), fileNamesToRemove.front());
```